File: RepoTransBench/target_projects/Java/Go/knightliao_pikaQ/tests/original/db_test.go
package original

import "testing"

const DB_NAME = "pikaqDemoWeb"

func TestDBNameConstant(t *testing.T) {
	if DB_NAME != "pikaqDemoWeb" {
		t.Errorf("DB.NAME = %s, expected pikaqDemoWeb", DB_NAME)
	}
}

File: RepoTransBench/target_projects/Java/C#/knightliao_pikaQ/tests/original/DBTest.cs
using Xunit;

namespace KnightliaoPikaQ.Tests.Original
{
    // Simulate DB class used in tests (with constant)
    public static class DB
    {
        public const string DB_NAME = "pikaqDemoWeb";
    }

    public class DBTest
    {
        [Fact]
        public void TestDBNameConstant()
        {
            Assert.Equal("pikaqDemoWeb", DB.DB_NAME);
        }
    }
}